src/interface/src/app/treatments/treatment-map/treatment-map.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Component, Input } from '@angular/core';
+import { JsonPipe, NgForOf } from '@angular/common';
+import {
+  DraggableDirective,
+  FeatureComponent,
+  GeoJSONSourceComponent,
+  LayerComponent,
+  MapComponent,
+  VectorSourceComponent,
+} from '@maplibre/ngx-maplibre-gl';
+
+import { Map as MapLibreMap, MapMouseEvent } from 'maplibre-gl';
+import { MapStandsComponent } from '../map-stands/map-stands.component';
+import { MapRectangleComponent } from '../map-rectangle/map-rectangle.component';
+import { SelectedStandsState } from './selected-stands.state';
+import { MapControlsComponent } from '../map-controls/map-controls.component';
+import { environment } from '../../../environments/environment';
+
+@Component({
+  selector: 'app-treatment-map',
+  standalone: true,
+  imports: [
+    JsonPipe,
+    NgForOf,
+    MapComponent,
+    VectorSourceComponent,
+    LayerComponent,
+    FeatureComponent,
+    DraggableDirective,
+    GeoJSONSourceComponent,
+    MapStandsComponent,
+    MapRectangleComponent,
+    MapControlsComponent,
+  ],
+  providers: [SelectedStandsState],
+  templateUrl: './treatment-map.component.html',
+  styleUrl: './treatment-map.component.scss',
+})
+export class TreatmentMapComponent {
+  mapLibreMap!: MapLibreMap;
+  readonly key = environment.stadiamaps_key;
+
+  // ew, not sure if I want to do prop drilling here or set up something else (provider?)
```

Comment:
I think this is okay for now, but maybe we can talk about what a provider could look like, especially if we're talking about where and how we're storing these and presenting them to the frontend?

Reply:
sure! I also think its ok now, and i don't want to prematurely optimize until we have this a little bit more developed.
A provider here can be something similar to what we have for the stands (SelectedStandsState).

Will update the comment though that one was mostly for me :D